godot-macros/src/lib.rs

```diff
@@ -932,6 +932,44 @@ pub fn derive_godot_class(input: TokenStream) -> TokenStream {
 ///     pub fn two(&self) { }
 /// }
 /// ```
+///
+/// `#[signal]` and `#[rpc]` attributes are not currently supported in secondary `impl` blocks.
+///
+///```compile_fail
+/// # use godot::prelude::*;
+/// #[derive(GodotClass)]
+/// #[class(init, base=Node)]
+/// pub struct MyNode {
+///     base: Base<Node>
+/// }
+///
+/// # #[godot_api]
+/// # impl MyNode {}
+///
```

Comment:
Is this still needed as a workaround?

Also, the `#` should span the next empty line as well, otherwise you have 2 empty ones in between -- see PR-generated docs